lib/router/index.js

```diff
@@ -6,7 +6,7 @@ let router = null
 const SingletonRouter = {}
 
 // Create public properties and methods of the router in the SingletonRouter
-const propertyFields = ['route', 'components', 'pathname', 'query']
+const propertyFields = ['components', 'pathname', 'route', 'query']
```

Comment:
@nkzawa I think we still need to keep these because they are pretty useful when doing custom routing.

If used for rendering, it'll throw an error in the SSR process.
So, I think that's fine.